packages/ozulip/ozulip.0.1/opam

```diff
@@ -0,0 +1,46 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "OCaml bindings to Zulip API"
+maintainer: ["Dario Pinto <dario.pinto@ocamlpro.com>"]
+authors: ["Mohamed Hernouf <mohamed.hernouf@ocamlpro.com>"]
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+tags: ["zulip" "bindings" "api"]
+homepage: "https://github.com/OCamlPro/ozulip"
+doc: "https://ocamlpro.github.io/ozulip/"
+bug-reports: "https://github.com/OCamlPro/ozulip/issues"
+depends: [
+  "ocaml" {>= "4.10"}
+  "dune" {>= "2.0"}
+  "ez_api" {>= "2.0.0"}
+  "re"
+  "base64"
+  "json-data-encoding" {>= "1.0.0"}
+  "logs"
+  "lwt"
```

Comment:
```suggestion
  "lwt" {>= "5.4.0"}
```
due to
```
#=== ERROR while compiling ozulip.0.1 =========================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.10.2 | pinned(https://github.com/OCamlPro/ozulip/archive/refs/tags/0.1.tar.gz)
# path                 ~/.opam/4.10/.opam-switch/build/ozulip.0.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p ozulip -j 47 @install
# exit-code            1
# env-file             ~/.opam/log/ozulip-7-5a734e.env
# output-file          ~/.opam/log/ozulip-7-5a734e.out
### output ###
# File "/home/opam/.opam/4.10/lib/biniou/biniou.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# (cd _build/default && /home/opam/.opam/4.10/bin/ocamlc.opt -w -40 -w -27-33 -g -bin-annot -I lib/.ozulip.objs/byte -I /home/opam/.opam/4.10/lib/astring -I /home/opam/.opam/4.10/lib/base/caml -I /home/opam/.opam/4.10/lib/base64 -I /home/opam/.opam/4.10/lib/biniou -I /home/opam/.opam/4.10/lib/cohttp -I /home/opam/.opam/4.10/lib/cohttp-lwt -I /home/opam/.opam/4.10/lib/cohttp-lwt-unix -I /home/opam/.opam/4.10/lib/conduit -I /home/opam/.opam/4.10/lib/conduit-lwt -I /home/opam/.opam/4.10/lib/conduit-lwt-unix -I /home/opam/.opam/4.10/lib/cstruct -I /home/opam/.opam/4.10/lib/domain-name -I /home/opam/.opam/4.10/lib/easy-format -I /home/opam/.opam/4.10/lib/ez_api -I /home/opam/.opam/4.10/lib/ez_api/cohttp_base -I /home/opam/.opam/4.10/lib/ez_api/cohttp_lwt -I /home/opam/.opam/4.10/lib/ez_api/debug -I /home/opam/.opam/4.10/lib/ez_api/encoding -I /home/opam/.opam/4.10/lib/ez_api/ezjsonm -I /home/opam/.opam/4.10/lib/ez_api/lwt_sys -I /home/opam/.opam/4.10/lib/ez_api/req_lwt_s -I /home/opam/.opam/4.10/lib/ez_api/request_lwt -I /home/opam/.opam/4.10/lib/ez_api/request_verbose -I /home/opam/.opam/4.10/lib/ez_file -I /home/opam/.opam/4.10/lib/fmt -I /home/opam/.opam/4.10/lib/hex -I /home/opam/.opam/4.10/lib/ipaddr -I /home/opam/.opam/4.10/lib/ipaddr-sexp -I /home/opam/.opam/4.10/lib/ipaddr/unix -I /home/opam/.opam/4.10/lib/json-data-encoding -I /home/opam/.opam/4.10/lib/json-data-encoding/stdlib -I /home/opam/.opam/4.10/lib/logs -I /home/opam/.opam/4.10/lib/lwt -I /home/opam/.opam/4.10/lib/lwt/unix -I /home/opam/.opam/4.10/lib/macaddr -I /home/opam/.opam/4.10/lib/magic-mime -I /home/opam/.opam/4.10/lib/mmap -I /home/opam/.opam/4.10/lib/ocaml/threads -I /home/opam/.opam/4.10/lib/ocplib_stuff -I /home/opam/.opam/4.10/lib/parsexp -I /home/opam/.opam/4.10/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.10/lib/re -I /home/opam/.opam/4.10/lib/re/posix -I /home/opam/.opam/4.10/lib/result -I /home/opam/.opam/4.10/lib/seq -I /home/opam/.opam/4.10/lib/sexplib -I /home/opam/.opam/4.10/lib/sexplib0 -I /home/opam/.opam/4.10/lib/stdlib-shims -I /home/opam/.opam/4.10/lib/stringext -I /home/opam/.opam/4.10/lib/uchar -I /home/opam/.opam/4.10/lib/uri -I /home/opam/.opam/4.10/lib/uri-sexp -I /home/opam/.opam/4.10/lib/uri/services -I /home/opam/.opam/4.10/lib/uuidm -I /home/opam/.opam/4.10/lib/uutf -I /home/opam/.opam/4.10/lib/yojson -intf-suffix .ml -no-alias-deps -open Ozulip__ -o lib/.ozulip.objs/byte/ozulip__Events.cmo -c -impl lib/events.ml)
# File "lib/events.ml", line 379, characters 2-15:
# 379 |   Lwt.dont_wait (fun () -> register config >>= aux) (function
#         ^^^^^^^^^^^^^
# Error: Unbound value Lwt.dont_wait
# (cd _build/default && /home/opam/.opam/4.10/bin/ocamlopt.opt -w -40 -w -27-33 -g -I lib/.ozulip.objs/byte -I lib/.ozulip.objs/native -I /home/opam/.opam/4.10/lib/astring -I /home/opam/.opam/4.10/lib/base/caml -I /home/opam/.opam/4.10/lib/base64 -I /home/opam/.opam/4.10/lib/biniou -I /home/opam/.opam/4.10/lib/cohttp -I /home/opam/.opam/4.10/lib/cohttp-lwt -I /home/opam/.opam/4.10/lib/cohttp-lwt-unix -I /home/opam/.opam/4.10/lib/conduit -I /home/opam/.opam/4.10/lib/conduit-lwt -I /home/opam/.opam/4.10/lib/conduit-lwt-unix -I /home/opam/.opam/4.10/lib/cstruct -I /home/opam/.opam/4.10/lib/domain-name -I /home/opam/.opam/4.10/lib/easy-format -I /home/opam/.opam/4.10/lib/ez_api -I /home/opam/.opam/4.10/lib/ez_api/cohttp_base -I /home/opam/.opam/4.10/lib/ez_api/cohttp_lwt -I /home/opam/.opam/4.10/lib/ez_api/debug -I /home/opam/.opam/4.10/lib/ez_api/encoding -I /home/opam/.opam/4.10/lib/ez_api/ezjsonm -I /home/opam/.opam/4.10/lib/ez_api/lwt_sys -I /home/opam/.opam/4.10/lib/ez_api/req_lwt_s -I /home/opam/.opam/4.10/lib/ez_api/request_lwt -I /home/opam/.opam/4.10/lib/ez_api/request_verbose -I /home/opam/.opam/4.10/lib/ez_file -I /home/opam/.opam/4.10/lib/fmt -I /home/opam/.opam/4.10/lib/hex -I /home/opam/.opam/4.10/lib/ipaddr -I /home/opam/.opam/4.10/lib/ipaddr-sexp -I /home/opam/.opam/4.10/lib/ipaddr/unix -I /home/opam/.opam/4.10/lib/json-data-encoding -I /home/opam/.opam/4.10/lib/json-data-encoding/stdlib -I /home/opam/.opam/4.10/lib/logs -I /home/opam/.opam/4.10/lib/lwt -I /home/opam/.opam/4.10/lib/lwt/unix -I /home/opam/.opam/4.10/lib/macaddr -I /home/opam/.opam/4.10/lib/magic-mime -I /home/opam/.opam/4.10/lib/mmap -I /home/opam/.opam/4.10/lib/ocaml/threads -I /home/opam/.opam/4.10/lib/ocplib_stuff -I /home/opam/.opam/4.10/lib/parsexp -I /home/opam/.opam/4.10/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.10/lib/re -I /home/opam/.opam/4.10/lib/re/posix -I /home/opam/.opam/4.10/lib/result -I /home/opam/.opam/4.10/lib/seq -I /home/opam/.opam/4.10/lib/sexplib -I /home/opam/.opam/4.10/lib/sexplib0 -I /home/opam/.opam/4.10/lib/stdlib-shims -I /home/opam/.opam/4.10/lib/stringext -I /home/opam/.opam/4.10/lib/uchar -I /home/opam/.opam/4.10/lib/uri -I /home/opam/.opam/4.10/lib/uri-sexp -I /home/opam/.opam/4.10/lib/uri/services -I /home/opam/.opam/4.10/lib/uuidm -I /home/opam/.opam/4.10/lib/uutf -I /home/opam/.opam/4.10/lib/yojson -intf-suffix .ml -no-alias-deps -open Ozulip__ -o lib/.ozulip.objs/native/ozulip__Events.cmx -c -impl lib/events.ml)
# File "lib/events.ml", line 379, characters 2-15:
# 379 |   Lwt.dont_wait (fun () -> register config >>= aux) (function
#         ^^^^^^^^^^^^^
# Error: Unbound value Lwt.dont_wait
```